setup.cfg

```
@@ -67,6 +67,8 @@ google =
     gcloud-aio-bigquery
 snowflake =
     apache-airflow-providers-snowflake
+apache-livy =
```

```suggestion
apache.livy =
```

To be consistent with OSS Airflow, we need `apache.livy`, so users can run `pip install astronomer-providers[apache.livy]`

---

[Done](https://github.com/astronomer/astronomer-providers/pull/178/commits/e14f0f20071ab218728591a6d8d411f4a3d9ba67)